.github/workflows/ci.yml

```diff
@@ -22,17 +22,48 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Download dependencies
-        run: composer update --no-interaction --prefer-dist --optimize-autoloader --prefer-stable
+        run: composer update --no-interaction --optimize-autoloader --prefer-stable
 
       - name: Run tests
         run: ./vendor/bin/simple-phpunit
 
+  build8:
+    name: Build PHP 8
```

Comment:
Why can't we use the "build" job?

Reply:
Easier to separate it out to force PHP 9.5.